triehash/README.md

```diff
@@ -0,0 +1,2 @@
+This crate provides utility functions to validate and initialize tries using flexible input.
```

Comment:
Nitpick, rephrase, `flexible input` (it is too ambiguous IMO)

What about generic input?

Reply:
Yeah good pt. I think I'll just drop the whole last part.

Reply:
Sounds good to me!